Source/Core/Core/AchievementManager.cpp

```diff
@@ -901,6 +904,16 @@ const AchievementManager::NamedIconMap& AchievementManager::GetChallengeIcons()
   return m_active_challenges;
 }
 
+std::vector<AchievementManager::FormattedValue> AchievementManager::GetActiveLeaderboards() const
+{
+  std::vector<AchievementManager::FormattedValue> display_values;
+  for (int ix = 0; ix < MAX_DISPLAYED_LBOARDS && ix < m_active_leaderboards.size(); ix++)
```

Comment:
```suggestion
  for (u32 ix = 0; ix < MAX_DISPLAYED_LBOARDS && ix < m_active_leaderboards.size(); ix++)
```

Otherwise this is doing a signed/unsigned comparison in the loop comparison.